package-lock.json

```diff
@@ -1,8054 +1,8 @@
 {
     "name": "jest-coverage-report-action",
     "version": "1.0.0",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
I can see that the `package-lock.json` file has been modified. It looks like you are using an old version of `npm` (<7) which generates lockfile with version 1, so could you please install the latest `npm` version with `npm i -g npm@latest` and rerun the `npm install` script in that package to update package-lock up to the 2nd version?

Reply:
Alright!